Plugins/DD4hep/include/Acts/Plugins/DD4hep/DD4hepBinningHelpers.hpp

```diff
@@ -30,6 +30,28 @@ namespace Acts {
 static std::vector<std::tuple<std::string, BinningValue>> allowedBinnings = {
     {"x", binX}, {"y", binY}, {"z", binZ}, {"phi", binPhi}, {"r", binR}};
 
+/// Helper method to convert the string to binning value
+///
+/// @param binningString
+///
+/// @return a binningValue
+inline BinningValue stringToBinningValue(const std::string &binningString) {
```

Comment:
I feel like we have a helper like this somewhere already, but I don't remember where.

Reply:
We had (in the JSON writing), but I removed this in favour in nlohmann's auto-generation of this for enums.